elements/pl-graph/pl-graph.py

```diff
@@ -137,17 +175,27 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     # Get attribs
     element = lxml.html.fragment_fromstring(element_html)
     engine = pl.get_string_attrib(element, "engine", ENGINE_DEFAULT)
-    input_param = pl.get_string_attrib(
+
+    input_param_matrix = pl.get_string_attrib(
         element, "params-name-matrix", PARAMS_NAME_MATRIX_DEFAULT
     )
+
+    input_param_graph = pl.get_string_attrib(
+        element, "params-name-networkx", PARAMS_NAME_GRAPH_DEFAULT
+    )
+
     input_type = pl.get_string_attrib(element, "params-type", PARAMS_TYPE_DEFAULT)
```

Comment:
You should update the documentation for the `params-type` attribute to clarify that it can now be `networkx` as well.